dayton/spiders/reap_spider.py

```diff
@@ -56,11 +58,19 @@ def getTaxEligibility(self, response):
     def getPaymentPlan(self, response):
         sel = Selector(response)
         item = response.meta['item']
-        item['paymentplan'] = False
-        for paymentplan in sel.xpath('//*[@id="content_middle_wide"]/div/form/div/tr/td/font/font/b/text()').extract():
-            item['paymentplan'] = 'Unapplied Payments:' in paymentplan or item['paymentplan']
+        item['payment_plan'] = False
+        
+        for paymentXPath in sel.xpath('//*[@id="content_middle_wide"]/div[3]/form/table/tr'):
+            try:
```

Comment:
I'm about to go out of town on a 3 day concert thing (rock on the range).
When I get back I'll fix this. I'm not sure what I did wrong on commit, but
it was definitely functional when I was at CfD, lol.

Sorry about that!

On Fri, May 15, 2015 at 8:01 AM, David Best <notifications@github.com>
wrote:

> In dayton/spiders/reap_spider.py
> <https://github.com/codefordayton/scrapers/pull/7#discussion_r30403096>:
>
> > @@ -56,11 +58,19 @@ def getTaxEligibility(self, response):
> >      def getPaymentPlan(self, response):
> >          sel = Selector(response)
> >          item = response.meta['item']
> > -        item['paymentplan'] = False
> > -        for paymentplan in sel.xpath('//*[@id="content_middle_wide"]/div/form/div/tr/td/font/font/b/text()').extract():
> > -            item['paymentplan'] = 'Unapplied Payments:' in paymentplan or item['paymentplan']
> > +        item['payment_plan'] = False
> > +
> > +        for paymentXPath in sel.xpath('//*[@id="content_middle_wide"]/div[3]/form/table/tr'):
> > +            try:
>
> I got
> exceptions.ValueError: time data '($137.17)' does not match format
> '%m-%d-%Y'
> on property:
>
> http://mctreas.org/pymtsbusdate.cfm?parid=R72%2006703%200049&taxyr=2014&own1=MUNLIN%20ETHEL%20%20%20%20%20%20%20%20%20%20%20%20%20%20
> %20%20%20%20%20%20%20%20%20%20%20%20%20%20%0A`
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/codefordayton/scrapers/pull/7/files#r30403096>.
>